src/plugin/ipc/file/fileconnection.cpp

```diff
@@ -856,13 +860,17 @@ bool FileConnection::checkDup(int fd)
 
 int FileConnection::openFile()
 {
-  JASSERT(jalib::Filesystem::FileExists(_path)) (_path)
+  dmtcp::string phys_path_string = "";
+  const char *phys_path =  virtual_to_physical_path ?
+                           virtual_to_physical_path(_path.c_str(), phys_path_string):
+                           _path.c_str();
+  JASSERT(jalib::Filesystem::FileExists(phys_path_string)) (_path)
```

Comment:
@gc00 : (do you mean on line 867?) One problem is that `FileExists` takes a cpp string, not a `char *`. I think a solution could be to initialize `phys_path_string` to `_path` instead of `""`. do you agree?